mpt/core/src/main/java/org/apache/james/mpt/protocol/FileProtocolSessionBuilder.java

```diff
@@ -177,6 +178,11 @@ else if (next.startsWith(SESSION_TAG)) {
                     }
                     sessionNumber = Integer.parseInt(number);
                 }
+                else if (next.startsWith(TIMER)) {
+                    TimerCommand timerCommand = TimerCommand.from(next.substring(6, 11));
```

Comment:
I actually understood it very well , I find it smart, and I think we don't need more then this.

However you can write it this way : 

```
TimerCommand timerCommand = TimerCommand.from(next.substring(TIMER.length + 1,
    TIMER.length + 1 + COMMAND_LENGTH)); 
```

This way you : 
 - Give indications about "why these constants"
 - You explicitly state that commands for timer should be of length 5 (lucky you)
 - And better explain the structure of the string